.github/workflows/publish-to-npm.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     - uses: actions/checkout@v2
     - uses: actions/setup-node@v2.1.4
       with:
-        node-version: '12.x'
+        node-version: '15.x'
```

Comment:
Are we using node 15 as a way to get npm 7?

Reply:
Yeah, I figured we'd probably do this for component workflows and should run ci-tools tests in the same environment -- it's actually not necessary for the publish step. Would you prefer we stick to an LTS release? Not sure there's a straightforward way to configure the npm version with `setup-node` but we could run an extra install command...

Reply:
What we would need to do is ensure that the test workflow uses the versions of NodeJS that we state are supported.

In the build service we use the version of the NodeJS that we support and then install the version of npm we support.

https://github.com/Financial-Times/origami-build-service/blob/master/.github/workflows/test.yml#L8-L11

Reply:
I actually removed that commit. The only "test" we're running is prettier so there was no need